src/store/modules/dev.js

```diff
@@ -18,6 +18,8 @@ const state = {
 
     logs_ws_endpoint: "wss://l932c9n1r3.execute-api.us-east-1.amazonaws.com/prod",  // prod
     //logs_ws_endpoint: "wss://8t7v3l1ok5.execute-api.us-east-1.amazonaws.com/dev", // dev
+
+    quickanalysis_endpoint: "http://quickanalysis.photonranch.org",
```

Comment:
you'll probably want all these URLs to be passed in eventually and set by env variable or something in your deploy, so you can support dev deployments hitting dev versions of services eventually

Reply:
Good idea. I'll keep this in mind.